libnetwork/drivers/bridge/setup_ip_forwarding_test.go

```diff
@@ -39,15 +42,8 @@ func readCurrentIPForwardingSetting(t *testing.T) []byte {
 }
 
 func writeIPForwardingSetting(t *testing.T, chars []byte) {
```

Comment:
This utility is only used in a single test; possibly (with the other idea of having a utility for this), we could reuse that utility in the tests as well;

```go
_, err := configureIPForwarding(ipv4ForwardConf, true)
if err != nil {
    t.Fatalf(err)
}
```

Reply:
Done.